sale_order_secondary_unit/migrations/11.0.1.0.2/post-migration.py

```diff
@@ -0,0 +1,23 @@
+#  Copyright 2019 Tecnativa - Sergio Teruel
+#  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    # Remove old security group_uom from product template view because now has
+    # a new group for secondary units
+    group_uom = env.ref('product.group_uom')
+    view_product_template = env.ref(
+        'sale_order_secondary_unit.product_template_form_view')
+    view_product_template.write({
+        'groups_id': [(3, group_uom.id)]
```

Comment:
Isn't enough with putting in the XML `groups_id` empty?

Reply:
I had two options, one is puting in the xml groups_id = [(5,0), (4, ref('new_group'))] and the other in a post_migration remove the old security group. I have chosen the second.

Reply:
But this technique on views seems a bit weird, and most being noupdate=0. Even more, `groups_id` in views doesn't use `(4,` terminology if I don't remember bad, but directly use the ref of the group.

Reply:
Let's continue this way as I'm not sure.